playground/package.json

```diff
@@ -2,74 +2,36 @@
   "name": "merchant-center-starter-app",
   "version": "1.0.0",
   "description": "A starter example for building a Merchant Center application",
-  "main": "index.js",
   "scripts": {
-    "commitmsg": "commitlint -e $GIT_PARAMS",
-    "precommit": "lint-staged",
     "build": "mc-scripts build",
     "start": "mc-scripts start",
     "start:prod": "mc-http-server --config $(pwd)/env.prod.json --csp $(pwd)/csp.json --use-local-assets",
     "now-build": "NODE_ENV=production npm run build",
-    "now-start": "NODE_ENV=production npm run start:prod",
-    "lint": "jest --projects jest.eslint.config.js",
-    "format": "npm run format:js && npm run format:md",
-    "format:js": "prettier --write 'src/**/*.js'",
-    "format:md": "prettier --parser markdown --write '*.md,src/**/*.md'",
-    "test": "jest --config jest.test.config.js",
-    "test:watch": "jest --config jest.test.config.js --watch",
-    "ci": "jest --projects jest.*.config.js"
+    "now-start": "NODE_ENV=production npm run start:prod"
   },
   "dependencies": {
-    "@commercetools-frontend/actions-global": "1.0.0-beta.31",
-    "@commercetools-frontend/application-shell": "1.0.0-beta.31",
-    "@commercetools-frontend/constants": "1.0.0-beta.31",
-    "@commercetools-frontend/i18n": "1.0.0-beta.31",
-    "@commercetools-frontend/l10n": "1.0.0-beta.31",
-    "@commercetools-frontend/mc-http-server": "1.0.0-beta.31",
-    "@commercetools-frontend/permissions": "1.0.0-beta.31",
-    "@commercetools-frontend/sdk": "1.0.0-beta.31",
-    "@commercetools-frontend/ui-kit": "2.0.0-rc.4",
+    "@commercetools-frontend/actions-global": "1.0.0-beta.35",
+    "@commercetools-frontend/application-shell": "1.0.0-beta.35",
+    "@commercetools-frontend/constants": "1.0.0-beta.35",
+    "@commercetools-frontend/i18n": "1.0.0-beta.35",
+    "@commercetools-frontend/l10n": "1.0.0-beta.35",
+    "@commercetools-frontend/mc-http-server": "1.0.0-beta.35",
+    "@commercetools-frontend/permissions": "1.0.0-beta.35",
+    "@commercetools-frontend/sdk": "1.0.0-beta.35",
+    "@commercetools-frontend/ui-kit": "2.0.0-rc.11",
+    "moment": "2.22.2",
+    "moment-timezone": "0.5.21",
     "prop-types": "15.6.2",
-    "react": "16.5.1",
-    "react-dom": "16.5.1",
+    "react": "16.5.2",
+    "react-dom": "16.5.2",
+    "react-intl": "2.7.1",
     "react-loadable": "5.5.0",
     "react-redux": "5.0.7",
     "react-router": "4.3.1",
-    "react-router-dom": "4.3.1"
+    "react-router-dom": "4.3.1",
+    "react-select": "1.2.1"
```

Comment:
Unrelated: Oh, I wasn't aware that we're using `react-select` here! We probably need to upgrade it to v2 at some point.

Reply:
Yeah, actually for the playground app it's because of the import

```js
import 'react-select/dist/react-select.css';
```

Also the AppShell still uses `v1`